ci/do_ci.sh

```diff
@@ -238,20 +243,31 @@ case $CI_TARGET in
         echo "  build options: ${BAZEL_BUILD_OPTIONS[*]}"
         echo "  release options:  ${BAZEL_RELEASE_OPTIONS[*]}"
 
-        bazel_with_collection test "${BAZEL_BUILD_OPTIONS[@]}" --remote_download_minimal "${BAZEL_RELEASE_OPTIONS[@]}" "${TEST_TARGETS[@]}"
+        bazel_with_collection \
+            test "${BAZEL_BUILD_OPTIONS[@]}" \
+            --remote_download_minimal \
+            "${BAZEL_RELEASE_OPTIONS[@]}" \
+            "${TEST_TARGETS[@]}"
 
         # Build release binaries
-        bazel build "${BAZEL_BUILD_OPTIONS[@]}" "${BAZEL_RELEASE_OPTIONS[@]}" //distribution/binary:release
+        bazel build "${BAZEL_BUILD_OPTIONS[@]}" "${BAZEL_RELEASE_OPTIONS[@]}" \
+              //distribution/binary:release
```

Comment:
is there a baisc rule about this re-formating? length or some other rules?

If let me do this, may be it would be:

```
        bazel_with_collection test\
            "${BAZEL_BUILD_OPTIONS[@]}" \
            --remote_download_minimal \
            "${BAZEL_RELEASE_OPTIONS[@]}" \
            "${TEST_TARGETS[@]}"


        bazel build \
            "${BAZEL_BUILD_OPTIONS[@]}" \
            "${BAZEL_RELEASE_OPTIONS[@]}" \
            //distribution/binary:release
```


I have no strong point to this. But if we hare some basic rules here, than we can make the style more unified.

Reply:
i think we cant really make rules for bash beyond those enforced by shellcheck

we can have some policies - like avoid overly long lines, dont use a boolean pseudo-ternary for conditions etc, but even then bash doesnt lend itself well to being overly proscriptive

wrt these - i followed a bit of a policy - but mostly my goal was to shift the args <--

re the build options specifically - im thinking to remove these altogether and just insert them in https://github.com/envoyproxy/envoy/blob/5e834d0d8d18918beddd4a8f1267f9fda8b49807/ci/build_setup.sh#L107-L110

this is step towards that, and also im trying to think of a ~clean way to do it